Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/ElmJsonLibraryReader.java

```diff
@@ -15,7 +16,7 @@ private ElmJsonLibraryReader() {
     public static JAXBContext getJaxbContext() {
         if (jaxbContext == null) {
             try {
-                jaxbContext = JAXBContext.newInstance(Library.class, Annotation.class);
+                jaxbContext = JAXBContext.newInstance(Library.class, Annotation.class, TypeSpecifier.class);
             } catch (JAXBException e) {
```

Comment:
I don't think we need to do anything like this here, the ELM deserialization should just proceed as normal.